ampcrowd/basecrowd/urls.py

```diff
@@ -2,9 +2,10 @@
 
 from basecrowd import views
 
-urlpatterns = patterns('',
+urlpatterns = patterns(
```

Comment:
Nitpick, but I believe it's considered good Django style to leave the first argument on the top line, because it's not a url like the subsequent arguments. Technically that's bad python style though, so I'm fine either way.